File: зад3.swift
func Re(number: Int) -> Int {
var result = 0;
var x = number;
//Переворачиваем
while x != 0 {
    result = result * 10 + (x % 10);
    //Деление без остатка
    x = Int(x / 10);
}
return result;
}

print("Введите количество: ")
var  n = Int(readLine()!)!

 //Ввод чисел
for i in 0...n-1 {
    print("Введите число: ");
    var  number = Int(readLine()!)!
    //Преравниваем
    number=Re(number: number);
   print(number);
}

File: зад3.cs
using System;

class MainClass {
    
 public static int Re(int number) {
    int result = 0;
    //Переворачиваем
    while (number != 0) {
        result = result * 10 + (number % 10);
        //Деление без остатка
        number= number / 10;
    }
    return result;
 }
  static void Main() {
    Console.WriteLine("Введите количество: ");
    int n = Convert.ToInt32(Console.ReadLine());
    //Ввод чисел
    for (int i = 0; i < n; i++) {
        Console.WriteLine("Введите число: ");
        int number = Convert.ToInt32(Console.ReadLine());
        //Преравниваем
        number=Re(number);
        Console.WriteLine(number);
    }
  }
}
